lib/suse/connect/zypper.rb

```diff
@@ -20,7 +20,7 @@ class << self
         # Returns an array of all installed products, in which every product is
         # presented as a hash.
         def installed_products
-          zypper_out = call('--xmlout --non-interactive products -i', false)
+          zypper_out = call('--no-refresh --xmlout --non-interactive products -i', false)
```

Comment:
Error: Invalid system credentials, probably because the registered system was deleted in SUSE Customer Center. Check https://scc.suse.com whether your system appears there. If it does not, please call SUSEConnect --cleanup and re-register this system.